src/main/thrift/parquet.thrift

```diff
@@ -932,27 +942,30 @@ struct FileMetaData {
   7: optional list<ColumnOrder> column_orders
   
   /** 
-   * Encryption algorithm. Note that this field is only used for files
-   * with plaintext footer. Files with encrypted footer store the algorithm id
+   * Encryption algorithm. This field is set only in encrypted files
+   * with plaintext footer. Files with encrypted footer store algorithm id
    * in FileCryptoMetaData structure.
    */
   8: optional EncryptionAlgorithm encryption_algorithm
+
+  /** 
+   * Retrieval metadata of key used for signing the footer. 
+   * Used only in encrypted files with plaintext footer. 
+   */ 
+  9: optional binary footer_signing_key_metadata
```

Comment:
This is not mentioned in the diagram 'Plaintext footer mode' of the design.

Reply:
mentioned in
https://github.com/apache/parquet-format/blob/encryption/Encryption.md